v10-compatibility

elizeuangelo opened the pull request:
I updated the module for v10 compatibility

elizeuangelo committed:
v10-compatibility

sasquach45932 commented:
Could create a release and add the manifest URL to this thread, please? It would be easier to install the V10 version from your fork. It seems the original github repo abandoned.
Thanks in advance.

elizeuangelo commented:
v10 manifest: https://raw.githubusercontent.com/elizeuangelo/better-rolltables/v10-release/module.json

This is a temporary manifest, by using this manifest you wont be getting new updates on the module, in case ultrakorne revives it.

sasquach45932 commented:
Thanks, I could install it. Unfrotunately rolling from embedded rolltables is not working. It just displays the rolltable name not the rolled item. Fixed with pull request.  [Please check/approve.](https://github.com/elizeuangelo/better-rolltables/pull/1)

sasquach45932 committed:
Update brt-builder.js

Fixing issue no roll on embedded rolltables.

sasquach45932 committed:
Update loot-chat-card.js

Fixing console warning.

sasquach45932 committed:
Update brt-builder.js

Previous pull fixes only when rolltable embedded from compendium, now it works when rolltable embedded from rolltables.

sasquach45932 committed:
Update brt-builder.js

sasquach45932 committed:
Update better-tables.js

Missing toolip for Share button
Warning in console during share

sasquach45932 committed:
Update settingsConfig.js

Fixed issue with  Advanced Settings:
- on click  tab pages did not contain any data, 
- save was not possible
- text were missing in some cases

sasquach45932 committed:
Update welcome-screen.html

sasquach45932 committed:
Update welcome-screen.html

sasquach45932 committed:
Update better-tables.js

Now it is wrting the Shared to the chatcard after sharing.

sasquach45932 committed:
Update better-tables.js

Fix for rolling from rolltable contextmenu failed

sasquach45932 committed:
Update init.js

Fix for context menues not displayed

sasquach45932 committed:
Update settingsConfig.js

Fix for context menues not displayed

sasquach45932 committed:
Update loot-chat-card.hbs

Fixed issue: rolling from Compendium links are broken

sasquach45932 committed:
Update API.js

Fixed issue: rolling from Compendium creates additional createChatMessage

sasquach45932 committed:
Update better-tables.js

Fixed issue: rolling from Compendium failed

sasquach45932 committed:
Update API.js

Variable name rename

sasquach45932 committed:
Update init.js

Fixed: Missing dice icon on Journal page

sasquach45932 committed:
Update better-tables.js

Fixed: Missing dice icon on Journal page

sasquach45932 committed:
Update better-tables.js

Fixed: Duplicated die icon on Journal page when item referring to compendium

elizeuangelo committed:
fixes inroll tables

elizeuangelo committed:
fixes inroll tables

elizeuangelo committed:
Merge branch 'master' of https://github.com/elizeuangelo/better-rolltables

elizeuangelo committed:
fixes inroll tables

elizeuangelo committed:
Merge branch 'master' of https://github.com/elizeuangelo/better-rolltables

elizeuangelo committed:
new version

elizeuangelo committed:
Merge pull request #1 from sasquach45932/master

Additional V10 Fixes

elizeuangelo commented:
New v10 version... need to be updated manually by importing the zip file.

**https://github.com/elizeuangelo/better-rolltables/releases/tag/v1.9.01**
